recipe/meta.yaml

```
@@ -161,6 +173,232 @@ outputs:
         - test -f $PREFIX/include/mkl.h  # [unix]
         - if not exist %LIBRARY_INC%\\mkl.h exit 1  # [win]
 
+  - name: onemkl-sycl-blas
+    version: {{ mkl_version }}
+    script: repack.sh   # [unix]
+    script: repack.bat  # [win]
+    number: {{ mkl_buildnum|int + dstbuildnum|int }}
+    requirements:
+      run:
+        - {{ pin_subpackage('mkl', exact=True) }}
+        - {{ pin_subpackage('dpcpp-cpp-rt', exact=True) }}
```

```suggestion
        - dpcpp-cpp-rt {{ mkl_version.split('.')[0] }}.*
```

And same should be applied for others